### PR TITLE
[17.0][MIG] base_partner_company_group: Migration to 17.0

### PR DESCRIPTION
**Pull request by MohamedOsman7**
This is a follow-up PR for this one: #1688

### CODE REVIEW

File: base_partner_company_group/views/contact_view.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <record id="action_open_group_members" model="ir.actions.act_window">
+        <field name="name">Company group members</field>
+        <field name="res_model">res.partner</field>
+        <field name="binding_model_id" ref="model_res_partner" />
+        <field name="view_mode">tree</field>
+        <field name="target">current</field>
+        <field
+            name="domain"
+        >[('company_group_id', 'in', context.get('active_ids'))]</field>
+    </record>
```
Comment (CRogos) [inline base_partner_company_group/views/contact_view.xml:56]: Not sure if we need to replace active_ids

**Review by CRogos — APPROVED** [review]
Code + functional LGTM
But maybe we can improve this module?
@pedrobaeza shall we move this? Otherwise you can merge it. same changes as origin PR but with cleaned history. 
image

**Comment by pedrobaeza** [comment]
Yes, it can be an option.

**Comment by CRogos** [comment]
@arquero20022 , 
@ConesaJM could you review?

**Review by christian-ramos-tecnativa — APPROVED** [review]
LGTM

**Comment by CRogos** [comment]
@pedrobaeza could you merge?

**Review by pedroguirao — APPROVED** [review]
LGTM

**Pull request merged**